src/Object/Image.php

```diff
@@ -230,9 +230,13 @@ public function getImage()
 		}
 
 		if ($this->isImagick()) {
-			/* Clean it */
-			$this->image = $this->image->deconstructImages();
-			return $this->image;
+			try {
+				/* Clean it */
+				$this->image = $this->image->deconstructImages();
+				return $this->image;
+			} catch (Exception $e) {
+				return false;
```

Comment:
I believe we should log something here.

Reply:
We haven't logged similar stuff in this class as well.